api-tests/dev_apis/crypto/test_c071/test_c071.c

```diff
@@ -124,8 +124,7 @@ int32_t psa_pake_set_user_test(caller_security_t caller __UNUSED)
       /* Expect a bad state when psa_pake_set_user is called on aborted inactive operation object */
```

Comment:
To maintain consistency with other test cases, check for bad state can be added as
```
if( check1[i].expected_status == PSA_SUCCESS)
{
    status = val->crypto_function(VAL_CRYPTO_PAKE_SET_USER, &operation, check1[i].user_id, check1[i].user_id_len);                                           
}
```